community/knative/charts/build/templates/build-crb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
```

Comment:
Missing app, chart, heritage, and release metadata labels; refer to [Helm's best practices for more](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)

Reply:
We will fix this in next commit